parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ColumnChunkPageWriteStore.java

```diff
@@ -215,15 +226,33 @@ public void writeDictionaryPage(DictionaryPage dictionaryPage) throws IOExceptio
     public String memUsageString(String prefix) {
       return buf.memUsageString(prefix + " ColumnChunkPageWriter");
     }
+
+    @Override
+    public ByteBufferAllocator getAllocator() {
+      return allocator;
+    }
```

Comment:
I still don't see why we have a getter for this. Things that need an allocator should get one injected.